package.json

```diff
@@ -1,42 +1,25 @@
 {
-  "name": "ccsseraphini",
-  "dependencies": {
-    "@chakra-ui/react": "^1.7.4",
-    "@emotion/react": "^11",
-    "@emotion/styled": "^11",
-    "framer-motion": "^5",
-    "next": "12.0.8",
-    "react": "17.0.2",
-    "react-dom": "17.0.2",
-    "react-icons": "^4.3.1",
-    "styled-components": "^5.3.3"
+  "private": true,
+  "scripts": {
+    "prepare": "husky install"
   },
+  "workspaces": [
+    "apps/*"
```

Comment:
what about `packages` for folder name?

Reply:
interop with vercel's `turbo` monorepo orchestrator, convention is `apps/`

Reply:
we could have another folder `packages/*` with utilities, configs or ui components etc

```
| apps
| -- web
| packages
| -- ui
| ---- hooks
| ---- containers
| -- jest
| ---- react
| ------ render-utils
| ---- node
| ------ fixtures
```

and anywhere in `apps/` we could do `import X from '@ui/hooks` etc